tests/sigrok-conv.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+
+if [ -z "$1" ] ; then
+  echo input file missing
+  echo "Usage: $0 input.cu8 output.sr [sample rate in kHz]"
```

Comment:
Can we add a sample rate parser here if the rate is missing from the arguments? Newer recordings have this in the name.

Reply:
Definitely, yes. This is just a stub to get the pieces together. I'll also add float-format output and remove sox. I don't think I'll add any zip code though.